python/couler/couler/argo.py

```diff
@@ -514,6 +516,10 @@ def yaml():
     ts.extend(_templates.values())
 
     wf["spec"] = {"entrypoint": entrypoint, "templates": ts}
+
+    if _workflow_ttl_cleaned is not None:
```

Comment:
Just confused about how to config `_workflow_ttl_cleaned ` with `couler` command-line tool?

Reply:
command line, we do not have this. you can configure this in your workflow code.

Reply:
I see.